stdlib/experimental/http/http_experimental.go

```diff
@@ -0,0 +1,156 @@
+package http
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// maxResponseBody is the maximum response body we will read before just discarding
+// the rest. This allows sockets to be reused.
+const maxResponseBody = 512 * 1024 // 512 KB
+
+// http get mirrors the http post originally completed for alerts & notifications
+var get = values.NewFunction(
+	"get",
+	semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"url":          semantic.String,
+			"headers":      semantic.Tvar(1),
+			"responseType": semantic.String,
+		},
+		Required: []string{"url"},
+		Return:   semantic.NewObjectPolyType(map[string]semantic.PolyType{"statusCode": semantic.Int, "body": semantic.String, "response": semantic.String}, semantic.LabelSet{"status"}, semantic.LabelSet{"status", "body", "res"}),
```

Comment:
I would like to see the return type here simplified to just the `statusCode` and the `body` properties.

The `statusCode` as an int makes sense the `body` should be a `semantic.Bytes`. 

Additionally these should always be returned, therefore we can remove the `responseType` parameter.

Reply:
What about headers? I still see them as a valuable option.

Reply:
Do you need the headers right now? If not then I want to wait because we don't have a good way to represent the headers.